src/PublicKey.cs

```diff
@@ -2,6 +2,7 @@
 using AElf.Types;
 using BitcoinPubKey = NBitcoin.PubKey;
 using IPubKey = NBitcoin.IPubKey;
+// ReSharper disable ArrangeNamespaceBody
```

Comment:
just curious: is this comment for future reference?

Reply:
this is to tell rider code inpector to ignore an issue, this is issue is “The curly braces after "namespace" should be replaced with a semicolon”, but this can not be done as .net2.1 can not use this format.